CHANGELOG.md

```diff
@@ -20,9 +20,9 @@
 - Bump Native SDK from v0.7.17 to v0.7.19 ([#2578](https://github.com/getsentry/sentry-dart/pull/2578), [#2588](https://github.com/getsentry/sentry-dart/pull/2588))
   - [changelog](https://github.com/getsentry/sentry-native/blob/master/CHANGELOG.md#0719)
   - [diff](https://github.com/getsentry/sentry-native/compare/0.7.17...0.7.19)
-- Bump Android SDK from v7.20.0 to v7.20.1 ([#2593](https://github.com/getsentry/sentry-dart/pull/2593))
-  - [changelog](https://github.com/getsentry/sentry-java/blob/main/CHANGELOG.md#7201)
-  - [diff](https://github.com/getsentry/sentry-java/compare/7.20.0...7.20.1)
+- Bump Android SDK from v7.20.0 to v8.0.0 ([#2593](https://github.com/getsentry/sentry-dart/pull/2593), [#2596](https://github.com/getsentry/sentry-dart/pull/2596))
```

Comment:
This change is breaking because it drops support for Android API 19.

Background:
Flutter SDK has [also dropped support for API 19](https://docs.flutter.dev/release/breaking-changes/android-kitkat-deprecation) but only since version 3.22.0 and we currently support 3.0.0

Reply:
@vaind do we have any urgency to to work on v9 because of this? e.g is android replay going to keep continue support v7 - dunno how much we depend on further/future work by android replay

Reply:
With the android bump & [this requirement for performance improvemnts](https://github.com/getsentry/sentry-dart/issues/2411#issuecomment-2586620688), I'm thinking we should start working on 9.0. 
While I'm not aware of any issues that would require as to update the Android SDK at the moment, it may be risky to release GA in FLutter SDK v8 when we can't get further Android SDK updates (or rather we can but it would be painful), in case any issue arises in the future.

It also sounds like a good time to combine the replay GA release in a new major version.